projects/collection-editor-library/src/lib/components/manage-collaborator/manage-collaborator.component.ts

```diff
@@ -0,0 +1,265 @@
+import { Component, EventEmitter, Input, OnInit, Output, ViewChild } from '@angular/core';
+import { ToasterService } from '../../services/toaster/toaster.service';
+import { EditorService } from '../../services/editor/editor.service';
+import { HelperService } from '../../services/helper/helper.service';
+import { ConfigService } from '../../services/config/config.service';
+
+import * as _ from 'lodash-es';
+@Component({
+  selector: 'lib-manage-collaborator',
+  templateUrl: './manage-collaborator.component.html',
+  styleUrls: ['./manage-collaborator.component.scss']
+})
+export class ManageCollaboratorComponent implements OnInit {
+  @Output() modalDismissEmitter = new EventEmitter<any>();
+  @ViewChild('modal', { static: false }) private modal;
```

Comment:
`static: false` is remove in angular 9, right?

Reply:
removed static: false